Src/WitsmlExplorer.Api/Models/LogObject.cs

```diff
@@ -19,5 +19,9 @@ public class LogObject
         public DateTime DateTimeCreation { get; set; }
         public DateTime? DateTimeLastChange { get; set; }
         public string IndexCurve { get; set; }
+        public List<LogCurveInfo> LogCurveInfo { get; set; }
```

Comment:
These changes are not incorporated in other usages of this class (f.ex `LogObjectService`). There should be a singular way of reading log data and we might see some indication here that there is a mix of how to read data from log objects?